doc/building_macos.md

```diff
@@ -0,0 +1,102 @@
+# Building Julius for Mac OS
+
+## Installing the dependencies
+
+In order to be able to build Julius, you need to install the dependencies first.
+
+The dependencies are:
+
+* The command line developer tools;
```

Comment:
Trailing `;` for every item looks redundant: just end the bullet item without any punctuation.